### ISSUE TITLE
The footer

### ISSUE THREAD

**Issue by masterxamss** [masterxamss/trouve_ton_artisan#1]
- [x] The same footer on every page;
- [x] Menu for the legal pages (legal mentions, personal data, accessibility, cookies), these pages should be empty and will be filled in later by a specialized office;
- [x] Address and contact details of the agency in Lyon:
101 cours Charlemagne
CS 20033
69269 LYON CEDEX 02
France
+33 (0)4 26 73 40 00
